samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -1033,15 +1019,31 @@ static int bis_sync_req_cb(struct bt_conn *conn,
 			   const struct bt_bap_scan_delegator_recv_state *recv_state,
 			   const uint32_t bis_sync_req[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS])
 {
-	printk("BIS sync request received for %p: 0x%08x->0x%08x, broadcast id: 0x%06x, (%s)\n",
-	       recv_state, requested_bis_sync, bis_sync_req[0], recv_state->broadcast_id,
+	requested_bis_sync = 0;
+	requested_subgroup_sync = 0;
+
+	for (uint8_t subgroup = 0; subgroup < recv_state->num_subgroups; subgroup++) {
+		if (bis_sync_req[subgroup] != 0) {
+			/* bis_sync_req[subgroup] = 0b1 or BT_BAP_BIS_SYNC_NO_PREF */
```

Comment:
```suggestion
```

Not sure what the point of this comment is. It's obvious that `bis_sync_req[subgroup]` is not 0 from the code `if (bis_sync_req[subgroup] != 0) {`

Reply:
Yes. Comment not needed. Will delete it